lib/web/apiserver.go

```
@@ -610,14 +610,24 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		secondFactor = cap.GetSecondFactor()
 	}
 
+	// disable joining sessions if proxy session recording is enabled
+	var canJoinSessions = true
+	clsCfg, err := h.cfg.ProxyClient.GetClusterConfig()
```

don't ignore this error, return error

---

this is webConfig, we do not return errors for webConfig

---

why?

---

because this would be a script error not resulting in anything. 
also, we still would want web app to work even if 1 section of the config failed to initialize.

---

ok, that's fair